cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -175,6 +177,11 @@ func (command *Evaluate) Initialize(args []string) (evaluationContext *evaluate.
 			language.DefaultExecutionTimeout = time.Duration(command.ExecutionTimeout) * time.Minute
 		}
 
+		if command.RunIDOffset == 0 {
+			command.logger.Panicf("number of run-id-starts-at must be greater than zero")
+		}
```

Comment:
@Munsio Why?

Offset could also be zero for the first run, no?

Reply:
Yeah I just added it so it is there, would have probably found it myself later on.

Reply:
Removing that guard.